src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs

```diff
@@ -1490,13 +1479,15 @@ public void ReorderFiles(ImmutableArray<string> filePaths)
                         }
                     }
 
+                    var documentIds = filePaths.Select(x => _documentPathsToDocumentIds[x]).ToImmutableList();
```

Comment:
I wasn't sure if you wanted to switch the ContainsKey to a .TryGetValue() so you're only traversing it once, but meh.

Reply:
Yea, we can do that as well.